lib/src/mt_dev.c

```diff
@@ -1767,6 +1775,13 @@ struct mt_tx_queue* mt_dev_get_tx_queue(struct mtl_main_impl* impl, enum mtl_por
 
   mt_pthread_mutex_lock(&inf->tx_queues_mutex);
   for (uint16_t q = 0; q < inf->max_tx_queues; q++) {
+    if (inf->drv_type == MT_DRV_IGC && inf->tx_pacing_way == ST21_TX_PACING_WAY_TSN) {
+      if (is_st21_traffic) {
```

Comment:
Can you leave a comment in the code for why we need a `q == 0` judgement here?

Reply:
> Can you leave a comment in the code for why we need a `q == 0` judgement here?

Resolved by [commit](https://github.com/OpenVisualCloud/Media-Transport-Library/commit/a38006b72d6c437fdaf92916efdfe5173b8dbd76)